integration/container/windows_isolation_test.go

```diff
@@ -328,11 +328,10 @@ func TestWindowsProcessIsolationResourceConstraints(t *testing.T) {
 				})
 			}
 
-			// Special handling for CPU quota test case
-			if tc.name == "CPU quota constraint" {
+			// Special handling for CPU count test case
+			if tc.name == "CPU count constraint" {
 				opts = append(opts, func(config *container.TestContainerConfig) {
-					config.HostConfig.CPUQuota = 50000   // 50% of CPU
-					config.HostConfig.CPUPeriod = 100000 // Standard period
+					config.HostConfig.CPUCount = 2 // Limit container to 2 CPUs
 				})
```

Comment:
This is fixing up the previous commit; probably good to squash it with the previous one.